server_admin/topics/user-federation/sssd.adoc

```diff
@@ -95,69 +117,85 @@ This script makes the necessary changes to `/etc/sssd/sssd.conf`:
++
+If you do not have permission to access the SSSD service, ensure that the user running the {project_name} server is in the `/etc/sssd/sssd.conf` file in the following section:
++
+[source]
+----
+[ifp]
+allowed_uids = root, your_username
+----
 
 ==== Enabling the SSSD Federation Provider
 
-{project_name} uses DBus-Java to communicate at a low level with D-Bus, which depends on the http://www.matthew.ath.cx/projects/java/[Unix Sockets Library].
+{project_name} uses DBus-Java to communicate at a low level with D-Bus. D-Bus depends on the http://www.matthew.ath.cx/projects/java/[Unix Sockets Library].
 
 ifeval::[{project_community}==true]
 
-An RPM for this library can be found in https://github.com/keycloak/libunix-dbus-java/releases[this repository]. Before installing it, be sure to check the RPM signature:
+You can find an RPM for this library in https://github.com/keycloak/libunix-dbus-java/releases[the keycloak repository]. Before installing this RPM, check the RPM signature using this command:
 
+[source]
+----
   $ rpm -K libunix-dbus-java-0.8.0-1.fc24.x86_64.rpm
   libunix-dbus-java-0.8.0-1.fc24.x86_64.rpm:
     Header V4 RSA/SHA256 Signature, key ID 84dc9914: OK
     Header SHA1 digest: OK (d17bb7ebaa7a5304c1856ee4357c8ba4ec9c0b89)
     V4 RSA/SHA256 Signature, key ID 84dc9914: OK
     MD5 digest: OK (770c2e68d052cb4a4473e1e9fd8818cf)
-  $ sudo yum install libunix-dbus-java-0.8.0-1.fc24.x86_64.rpm
+----
+
+Install the RPM using this command:
+
+[source]
+----
+$ sudo yum install libunix-dbus-java-0.8.0-1.fc24.x86_64.rpm
+----
 
 endif::[]
 
 ifeval::[{project_product}==true]
 
-Before enabling the SSSD Federation provider, you must install the RPM for this library:
+Before enabling the SSSD Federation provider, install the RPM for this library:
 
-  $ sudo yum install rh-sso7-libunix-dbus-java
+[source]
+----
+$ sudo yum install rh-sso7-libunix-dbus-java
+----
 
 endif::[]
 
-For authentication with PAM {project_name} uses JNA. Be sure you have this package installed:
+{project_name} uses JNA to authenticate with PAM. Ensure you have the JAN package installed.
 
-  $ sudo yum install jna
+[source]
+----
+$ sudo yum install jna
 
-Use `sssctl user-checks` command to validate your setup:
+----
 
+Use the `sssctl user-checks` command to validate your setup:
+[source]
+----
   $ sudo sssctl user-checks admin -s keycloak
+----
 
 === Configuring a Federated SSSD Store
 
-After the installation, you need to configure a federated SSSD store.
+After the installation, you must configure a federated SSSD store.
 
-To configure a federated SSSD store, complete the following steps:
-
-. Navigate to the Administration Console.
-. From the left menu, select *User Federation.*
-. From the *Add Provider* dropdown list, select *sssd.* The sssd configuration page opens.
+.Procedure
+. From the left menu, click *User Federation*.
+. Click the *Add Provider* drop-down list.
```

Comment:
drop "drop-down*.
. From the  *Add Provider* list, select *sssd*.   (see https://docs.engineering.redhat.com/display/CCSENABLE/GUI+elements)  The *sssd* configuration page displays.  (see the bible)